draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1714,13 +1714,23 @@ still provide adequate output for incomplete logs.
 
 # Security and privacy considerations {#privacy}
 
-Protocols such as TLS {{!RFC8446}} and QUIC {{!RFC9000}} provide varying degrees
+Protocols such as TLS {{?RFC8446}} and QUIC {{?RFC9000}} provide varying degrees
 of secure protection for the wire image {{?RFC8546}}. There is inevitably
 tension between security and observability, when logging can reveal aspects of
 the wire image, that would ordinarily be protected. This tension equally applies
 to any privacy considerations that build on security properties, especially if
 data can be correlated across data sources.
 
+Any data that is determined to be necessary for a use case at hand could be
+logged or captured. As per {{?RFC6973}}, operators must be aware that such data
```

Comment:
This first sentence is not precise about whether it refers to the logging use case or the use case of the logged exchange.

Reply:
I agree. I'm going to track this in a separate ticket (along with your other mention of use-cases)